### ISSUE TITLE
JSDoc migration broke some type re-exports

### ISSUE THREAD

**Issue by MysteryBlokHed** [color-js/color.js#551]
The tests `type-accessibility.ts` and `type-fn-accessibility.ts` are currently both failing. The migration to JSDoc seems to have excluded some types that were defined/re-exported in the `index.d.ts` and `index-fn.d.ts` files. These should be re-added to preserve compatibility.

<https://github.com/color-js/color.js/blob/b6e92aa8bf396d7f106a950a240698011a724bce/types/test/type-accessibility.ts#L1-L37>

<https://github.com/color-js/color.js/blob/b6e92aa8bf396d7f106a950a240698011a724bce/types/test/type-fn-accessibility.ts#L1-L32>
